### ISSUE TITLE
Inserted and deleted items are not displayed

### ISSUE THREAD

**Issue by IgorAlbert** [retest/recheck.cli#111]
I opened the same report in review and in the cli. In review you could look at inserted/deleted elements and in the cli they were not displayed.

**Comment by beatngu13** [comment]
This is still an issue, which can be reproduced via recheck-web. Change the lines 93 – 95 in `src/test/resources/pages/simple-page.html` for `SimplePageIT` as follows:

```diff
-<span id="complexJsonText">{a="\\b\\\"'\'"}</span>
+<span id="changed">{a="\\b\\\"'\'"}</span>
 
-<span id="trimmedSpace">             ​‌‍  ⁠　test             ​‌‍  ⁠　</span>
+<div id="inserted">I have been inserted!</div>
```

This does the following:

1. Change: `id` attribute of the first `<span>` from `complexJsonText` to `changed`.
2. Deletion: `<span id="trimmedSpace">`
3. Insertion: `<div id="inserted">I have been inserted!</div>`

Invoking `diff` results in:

```
$ diff --exclude=positioning.filter de.retest.web.it.SimplePageIT.report
Checking test report in path '/path/to/de.retest.web.it.SimplePageIT.report'.

Test 'simple-page-ChromeDriver' has 3 difference(s) in 1 state(s):
open resulted in:
	span [{a="\\b\\\"'\'"}] at 'html[1]/body[1]/span[3]':
		id: expected="complexJsonText", actual="changed"
Test 'simple-page-FirefoxDriver' has 3 difference(s) in 1 state(s):
open resulted in:
	span [{a="\\b\\\"'\'"}] at 'html[1]/body[1]/span[3]':
		id: expected="complexJsonText", actual="changed"
```

Although 3 differences for each test are mentioned, only the change gets printed. This is likely an issue in `TestReportPrinter` (i.e. recheck) and related to https://github.com/retest/recheck-web/issues/295/. The corresponding workaround via https://github.com/retest/recheck/pull/343/ only fixes the assertion error.

CC @roesslerj @oezelenes

**Comment by Creepow** [comment]
Fixed with 1.5.0
